dts/riscv32/rv32m1.dtsi

```diff
@@ -282,7 +282,7 @@
 		};
 
 		flash-controller@40023000 {
-			compatible = "nxp,kinetis-ftfe";
+			compatible = "openisa,rv32m1-ftfe";
```

Comment:
I would follow the normal naming with:

fctrl0: flash-controller@40023000

Reply:
This issue is not included by current pull request. It is better to create a new one for this issue.